### PR TITLE
Simplified and removed dependencies of `to_padded_hex'

### PR DESCRIPTION
**Pull request by zv**
I wasn't able to run the unit tests locally immediately so either run the tests locally or let CI finish up before merging.

### CODE REVIEW

File: src/polyswarmd/utils.py [hunk 1]
```diff
@@ -226,24 +230,31 @@ def camel_case_to_snake_case(s):
     return re.sub(r'([a-z0-9])([A-Z])', r'\1_\2', s1).lower()
 
 
-def to_padded_hex(val):
-    if not g.chain:
-        raise ValueError('g.chain not found')
+def to_padded_hex(val: Union[str, bool, int, bytes]) -> str:
+    "Convert an argument to a hexadecimal string and pad zero-extend to 64"
```
Comment (mrtizmoatwork) [inline src/polyswarmd/utils.py]: use 3 quotes at start and end of the comment block

File: src/polyswarmd/utils.py [hunk 2]
```diff
@@ -226,24 +230,31 @@ def camel_case_to_snake_case(s):
...
-
-    l = 64 - len(padded_hex)
-
-    for i in range(0, l):
-        padded_hex = '0' + padded_hex
+        raise TypeError("Cannot convert to padded hex value")
 
-    return padded_hex
+    # This leaves open the possibility of returning len(result) > 64 chars.
+    return hex_suffix.rjust(64, '0')
```
Comment (mrtizmoatwork) [inline src/polyswarmd/utils.py]: how could this return a string having len() > 64?

Comment (zv) [reply src/polyswarmd/utils.py]: @mrtizmoatwork `to_padded_hex('0x' + '1' * 65)`

Comment (mrtizmoatwork) [reply src/polyswarmd/utils.py]: @Rizato what would happen if this function returned a string that is larger than 64 chars? So, for example. how would polyswarmd handle a nonce that is 65 or more digits long?

Comment (Rizato) [reply src/polyswarmd/utils.py]: I think we should raise some exception if the conversion is over 64, so we can return 400 in the offers route. Unfortunately the state blob needs everything exactly 64 length. 

If the state values weren't coming from users I would say let's ignore it.

Comment (Rizato) [reply src/polyswarmd/utils.py]: Alternatively, we improve the schema in the offers route to prevent too long input. 

I prefer that route.

Comment (zv) [reply src/polyswarmd/utils.py]: @mrtizmoatwork @Rizato I'm raising a `ValueError` if the result would be longer than 64 chars

**Pull request merged**